docs/content/preview/reference/drivers/ycql-client-drivers.md

```diff
@@ -16,6 +16,8 @@ The following client drivers are supported for use with the [Yugabyte Cloud Quer
 
 For tutorials on building a sample application with the following client drivers, click the relevant link included below for each driver.
 
+Note that for YCQL you have to use the Yugabyte-specific client drivers, otherwise you can run into `NotSupportedError: Unknown data type code 0x0080. Have to skip entire result set.` errors.
```

Comment:
this issue is not with most of the datatypes, I guess it is only with JSONB type, it is good to specify it clearly.

Reply:
Yes, for the JSONB datatype the original `cassandra` driver does not support select query. But it is supported in `yb-cassandra-driver`. @def- and I were discussing about it a few days ago and we had to remove the existing installation of the original `cassandra` driver and install the `yb-cassandra-driver` again to get rid of this issue.

Reply:
As discussed with Oleg, there are also other differences, can be slower or work incorrectly. Jsonb might just be the most obvious failure. So I think we should keep the recommendation general.